lib/graphql/plug/endpoint.ex

```diff
@@ -63,6 +66,22 @@ defmodule GraphQL.Plug.Endpoint do
     end
   end
 
+  defp evaluate_root_value(conn, {mod, func}) do
+    apply(mod, func, [conn])
+  end
+
+  defp evaluate_root_value(conn, root_value) when is_function(root_value, 1) do
```

Comment:
root_value isn't value here it's a function, call it `fn`, `get_root_value`, ...? think of something better than either of those... ;)